health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/downloadTemplate.js

```
@@ -0,0 +1,369 @@
+    const [allBoundaries1, setAllBoundaries1] = useState([]);
+    useEffect(()=>{
+        console.log("the heirarchy defautl defintion", data?.TenantBoundary?.[0]?.boundary);
+        // allBoundaries1 = data?.TenantBoundary?.[0]?.boundary
+        setAllBoundaries1(data?.TenantBoundary?.[0]?.boundary)
+    }, [data]);
+
+    const generatePaths = (data, path = [], allDatas) => {
+        const newPath = [...path, data.code];
+        allDatas.push(newPath);
+        if (data.children && data.children.length > 0) {
+            for (let child of data.children) {
```

Biome

[error] This let declares a variable that is only assigned once.

'child' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)